tests/Unit/Evolution/Systems/RelativisticEuler/Valencia/Test_PrimitiveFromConservative.cpp

```diff
@@ -0,0 +1,212 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cmath>
+#include <cstddef>
+#include <limits>
+#include <memory>
+#include <pup.h>
+#include <random>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/RelativisticEuler/Valencia/ConservativeFromPrimitive.hpp"
+#include "Evolution/Systems/RelativisticEuler/Valencia/PrimitiveFromConservative.hpp"
+#include "PointwiseFunctions/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "tests/Utilities/MakeWithRandomValues.hpp"
+
+// IWYU pragma: no_forward_declare EquationsOfState::EquationOfState
+
+namespace {
+
+template <typename DataType>
+Scalar<DataType> specific_enthalpy(
```

Comment:
promote these to free functions; question is where?  @nilsdeppe do you have any thoughts?

Reply:
I have a couple questions and my thoughts depend on the answers:
- Are we suggesting making them visible only to other tests?
- specific_enthalpy only works for relativistic systems. If it is used beyond this test that needs to be encoded.

If the answer is we only want other tests to use them them probably `tests/Unit/PointwiseFunctions/EquationsOfState/` (not sure what the "right" file name is). If used outside of tests (which I'm not sure is what we want to do, but could reasonably do so in some cases), then replace `tests/Unit` with `src`. They're directly tied to EoS computations so I think that's at least a good first place for them.